db/db_with_timestamp_basic_test.cc

```diff
@@ -588,6 +588,326 @@ TEST_F(DBBasicTestWithTimestamp, CompactDeletionWithTimestampMarkerToBottom) {
   Close();
 }
 
+class DataVisibilityTest : public DBBasicTestWithTimestampBase {
+ public:
+  DataVisibilityTest() : DBBasicTestWithTimestampBase("data_visibility_test") {}
+};
+
+// Application specifies timestamp but not snapshot.
+//           reader              writer
+//                               ts'=90
+//           ts=100
+//           seq=10
+//                               seq'=11
+//                               write finishes
+//         GetImpl(ts,seq)
+// It is OK to return <k, t1, s1> if ts>=t1 AND seq>=s1.
```

Comment:
I feel this could be made clearer by wording it differently, e.g. by emphasizing that the key-value should *not* be returned because of the (implicit) snapshot taken at seq 10. (Similarly for other test cases below.)

Reply:
I added a sentence to emphasize this case here and in other tests.